Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Services/StringRandomizerService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Linq;
+using MvvmSeed.Domain.Model;
+
+namespace MvvmSeed.Domain.Services
+{
+    /// <summary>
+    /// Simple implementation of <see cref="IStringRandomizerService"/> that stores randomization info on a local database (Realm)
+    /// </summary>
+    public class StringRandomizerService : IStringRandomizerService
+    {
+        private readonly LocalStorageContext _dbContext;
```

Comment:
As I said in my first review: there are different vocabulary applying in these scenarios. I think you should standardize it because it's kind of difficult for reading through all the classes (at least for me).

Look at this class for example. `LocalStorageContext` - `_dbContext` - `local database`.

I think it will read better if you use always the same wording. Also: this is a local storage approach where you don't care about being a db, file or remote service? If that is the case I believe you should not use the db wording.